examples/command.rs

```diff
@@ -81,6 +81,27 @@ fn setup(
                     },
                 );
             });
+        })
+        .with_child(|child| {
+            child.name(Cow::Borrowed("tp")).with_child(|child| {
+                child.name(Cow::Borrowed("tp.x")).parser::<i32>(Default::default()).with_child(|child| {
+                    child.name(Cow::Borrowed("tp.y")).parser::<i32>(Default::default()).with_child(|child| {
+                        child.name(Cow::Borrowed("tp.z")).parser::<i32>(Default::default())
```

Comment:
Could the gamemode command above be refactored to use a parser for the parameter?

Reply:
GameMode must implement `Parse` and `RawParseSuggestions` traits. I didn't do it yet because I want to deal with essential parts like suggestions handling and so on and then do all of brigadier's parsers.